model_compression_toolkit/core/exporter.py

```diff
@@ -90,3 +88,40 @@ def export_model(tg,
 
     return quantized_model, user_info
 
+
+def experimental_export_model(tg: Graph,
+                              fw_info: FrameworkInfo,
+                              fw_impl: FrameworkImplementation,
+                              tb_w: TensorboardWriter,
+                              bit_widths_config: List[int]):
+    """
+    A function for quantizing the graph's weights and build a quantized framework model from it.
+
+    Args:
+        tg: A prepared for quantization graph.
+        fw_info: Information needed for quantization about the specific framework (e.g., kernel channels indices,
+        groups of layers by how they should be quantized, etc.).
+        fw_impl: FrameworkImplementation object with a specific framework methods implementation.
+        tb_w: TensorBoardWriter object to log events.
+        bit_widths_config: mixed-precision bit configuration to be added to model user_info
+
+    Returns:
+        Quantized model in the input framework, and information the user may need in order to use the quantized model.
+    """
+
+    # tg = quantize_graph_weights(tg,
+    #                             fw_info=fw_info,
+    #                             fw_impl=fw_impl,
+    #                             fake_quant=False)
+    # if tb_w is not None:
+    #     tb_w.add_graph(tg, 'after_quantization')
+
+    quantized_tg = substitute(tg,
+                              fw_impl.get_substitutions_pre_build())
+
+    quantized_model, user_info = fw_impl.model_builder(quantized_tg,
```

Comment:
I suggest not using model_builder and maybe we should call it quantized_model_builder and use it as now API